src/library_pthread.js

```diff
@@ -277,18 +277,14 @@ var LibraryPThread = {
         } else if (cmd === 'loaded') {
           worker.loaded = true;
 #if ENVIRONMENT_MAY_BE_NODE
-          if (ENVIRONMENT_IS_NODE) {
+          if (ENVIRONMENT_IS_NODE && !worker.pthread_ptr) {
```

Comment:
Please add a comment here - I'm not sure offhand what it means to have or not have `pthread_ptr` defined.

Reply:
The comment below sort of covers it already ("& idle", aka doesn't have a pthread associated with it), but sure, I can elaborate a bit.